seacrowd/sea_datasets/xcopa/xcopa.py

```diff
@@ -45,12 +45,20 @@
 
 _HOMEPAGE = "https://github.com/cambridgeltl/xcopa"
 
-_LICENSE = "Unknown"
+_LICENSE = "Licenses.CC_BY_4_0.value"
```

Comment:
this is an Enum object, not a string

```suggestion
_LICENSE = Licenses.CC_BY_4_0.value
```

Reply:
added